analyzer/src/main/kotlin/managers/SpdxDocumentFile.kt

```diff
@@ -25,6 +25,7 @@ import java.util.SortedSet
 import org.ossreviewtoolkit.analyzer.AbstractPackageManagerFactory
 import org.ossreviewtoolkit.analyzer.PackageManager
 import org.ossreviewtoolkit.downloader.VersionControlSystem
+import org.ossreviewtoolkit.model.Hash
```

Comment:
Commit message: The title is too long, it should not exceed 75 chars. If you have more to say (which generally is good), then please put that into the commit message's body text.

Also the commit message should be prefix with the "scope" of the commit. In this case probably "SpdxDocumentFile: ...".